.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,8 +1,8 @@
 blank_issues_enabled: false
 contact_links:
   - name: Auth0 Community
-    url: https://community.auth0.com/c/sdks/5
+    url: https://community.auth0.com
     about: Discuss this SDK in the Auth0 Community forums
   - name: Library Documentation
     url: https://github.com/auth0/omniauth-auth0#documentation
-    about: Read the library docs on Auth0.com
+    about: Read the library docs
```

Comment:
Only 1 out of 3 resources in the link are in auth0.com

<img width="334" alt="Screenshot 2023-04-26 at 8 27 13 PM" src="https://user-images.githubusercontent.com/5055789/234723808-e33ee0c3-5b6c-404a-ab2e-479b70e8577a.png">